admin/src/pages/CreationConfirm.vue

```diff
@@ -116,6 +116,11 @@ export default {
       pageSize: 25,
     }
   },
+  watch: {
```

Comment:
Can we please have a unit test for at least one case?

Reply:
[c6bde11](https://github.com/gradido/gradido/pull/2902/commits/c6bde11de064eafcfa171e5a9198cdd3eb8edefa)